### PR TITLE
nautilus: osd: revamp {noup,nodown,noin,noout} related commands

### PR DESCRIPTION
**Pull request by xiexingguo**
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by smithfarm** [comment]
No release note needed for this?

**Comment by yuriw** [comment]
[wip-yuri6-testing-2019-06-10-1956-nautilus](https://trello.com/c/QGfZaPf9/635-wip-yuri6-testing-2019-06-10-1956-nautilus)

**Comment by neha-ojha** [comment]
> No release note needed for this?

@xiexingguo this might be a good idea, thoughts?

**Comment by yuriw** [comment]
@xiexingguo This PR passed the tests and as soon as @neha-ojha 's comment is addressed can be merged

**Comment by yuriw** [comment]
[wip-yuri-testing-2019-06-11-2200-nautilus](https://trello.com/c/WVOwNvcj/638-wip-yuri-testing-2019-06-11-2200-nautilus)

**Comment by xiexingguo** [comment]
> @xiexingguo this might be a good idea, thoughts?

Done.

File: PendingReleaseNotes [hunk 1]
```diff
@@ -7,3 +7,10 @@
   for more details on the background of this change.
 
 * Nautilus-based librbd clients can now open images on Jewel clusters.
+
```
Comment (neha-ojha) [inline PendingReleaseNotes]: I think we'll want to add a section for 14.2.2

Comment (xiexingguo) [reply PendingReleaseNotes]: Updated

**Pull request merged**